### PR TITLE
bring back the [PyPI] downloads badges

### LINKED ISSUE #1671: PyPi downloads badge showing "no longer available"
Currently, the PyPi downloads look like this:

But I would like to see the numbers of the downloads.
In some way, 

Examples: PyPI: 
-  `https://img.shields.io/pypi/dm/Django.svg`
-  `https://img.shields.io/pypi/dw/Django.svg`
-  `https://img.shields.io/pypi/dd/Django.svg`

But these badges work:
- PyPI - License `https://img.shields.io/pypi/l/Django.svg`

Source code:
- https://github.com/badges/shields/blob/b126b4ebdc64015a3d6e845d9c051f69ad81c4ea/server.js#L2250

### PR DESCRIPTION
**Pull request by chris48s**
..using [pypistats.org](https://pypistats.org/api/#api-packages-package-recent)

closes #1671

### CODE REVIEW

File: services/pypi/pypi-downloads.service.js [hunk 1]
```diff
@@ -1,12 +1,99 @@
...
+      url,
+      schema: pypiStatsSchema,
+      errorMessages: { 404: 'package not found' },
+    })
+  }
+
+  static render({ period, downloads }) {
+    return {
+      message: `${metric(downloads)}${periodMap[period].suffix}`,
+      color: downloads > 0 ? 'brightgreen' : 'red',
```
Comment (PyvesB) [inline services/pypi/pypi-downloads.service.js]: Would our `downloadCount` colour formatter be a good candidate here?

**Comment by chris48s** [comment]
@Natim - The reason you're still getting an error on that is because although this has been merged, it hasn't been deployed to shields.io yet. Unfortunately we've got a bit of a roadblock on getting stuff deployed at the moment: There are 2 maintainers who can push changes to production and both are absent so there is a bit of a backlog of stuff that is merged but not yet live.

**Comment by Natim** [comment]
Don't worry about it, I was just checking it :) Looking forward?

**Pull request merged**